pkg/utils/utils_test.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestAreMapsSameWhenEquivalent(t *testing.T) {
+	one := map[string]string{
+		"foo": "bar",
+		"xyz": "123",
+	}
+	two := map[string]string{
+		"xyz": "123",
+		"foo": "bar",
+	}
+	if !AreMapsSame(one, two) {
+		t.Errorf("Exp maps to evaluate to be equivalent - left: %v, right: %v", one, two)
+	}
+}
+func TestAreMapsSameWhenDifferent(t *testing.T) {
```

Comment:
Can we also test comparing two empty maps as well as an empty map compared to a non-empty map?